gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -354,6 +355,10 @@ message TripUpdate {
     // populated and will be ignored by consumers.
     // NOTE: This field is still experimental, and subject to change. It may be formally adopted in the future.
     optional string start_time = 3;
+    // Specifies the shape of the vehicle travel path of the trip. See definition of trips.shape_id in (CSV) GTFS.
+    // Refers to a shape defined in the (CSV) GTFS or in the (protobuf) real-time feed. 
+    // NOTE: This field is still experimental, and subject to change. It may be formally adopted in the future.
+  optional string shape_id = 8;
```

Comment:
This is set as 8 in the original PR: https://github.com/MobilityData/transit/pull/47/files#diff-a1a14af3b4aed7d10a3b167231b52fdb483cf71b88f1655ed2e28bccd18800ceR756

Wondering the impact of making this the next available value instead (4)?

Reply:
Interesting. I also didn't realize that the original PR existed. I think I agree with Rachel's assessment that the spec has changed so probably the initial PR won't merge, but we might want to give a nod to the initial PR when we submit this one and explain why we are just implementing a small subset right now.

That being said I think I would use the next available number here instead of 8.